exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,50 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class BaseAuditColumns:
+    TIMESTAMP = timestamp_column("LOG_TIMESTAMP", precision=3)
+    SESSION_ID = decimal_column("SESSION_ID", precision=20)
+    # RUN_ID must be obtained initially and remain unchanged during lifetime
+    # of AuditLogger. AuditLogger must inherit from QueryHandler and its first
+    # task is to get the RUN_ID.
+    # Proposed value: POSIX_TIME(SYSTIMESTAMP(9)) * 1000
+    RUN_ID = decimal_column("RUN_ID", precision=20)
+    ROWS_COUNT = decimal_column("ROWS_COUNT", precision=36)
+    # For ModifyQuery SPAN_TYPE is the Operation Name, e.g.
+    # CREATE_TABLE, INSERT, ERROR, COMMIT, CUSTOM:
+    SPAN_TYPE = varchar_column("SPAN_TYPE", size=128)
```

Comment:
Why is it "SPAN_TYPE" and not "OPERATION_TYPE"?

Reply:
Because Torsten changed the design to use SPAN_TYPE. See page "Audit Fields" in company wiki.

Reply:
@ahsimb and me currently agreed on the term "event". So it could be "`EVENT_TYPE`.